app/assets/javascripts/two-pane.js

```diff
@@ -144,6 +144,11 @@ function right_pane_content(response){
   var contentId;
   if (handle_redirect(response)) return; //session expired redirect to login
 
+  // move flash messages from response to main window
+  $('.flash', response).each(function(index, item) {
```

Comment:
I'm not sure I'm following, i think that .flash doesnt exists anymore on develop branch?